### PR TITLE
Find appt by name

### LINKED ISSUE #68: Search appointment by name

### PR DESCRIPTION
**Pull request by mechastriker3**
closes #68

### CODE REVIEW

**Review by edelinetenges — APPROVED** [review]
Just a few details that might need to be changed, but overall looks good!

File: src/main/java/seedu/address/logic/commands/FindAppointmentCommand.java [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 import seedu.address.model.Model;
+import seedu.address.model.appointment.Appointment;
+
+
 
 public class FindAppointmentCommand extends Command {
     public static final String COMMAND_WORD = "findAppt";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all Appointment whose names contain any of "
```
[the remark this answers is not in the document]
Comment (mechastriker3) [reply src/main/java/seedu/address/logic/commands/FindAppointmentCommand.java:17]: ok

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 2]
```diff
@@ -22,7 +20,7 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "the option " + PREFIX_OPTION + OPTION_TAG + " can be used to find all persons that match at least"
+            + "the Option " + PREFIX_OPTION + OPTION_TAG + " can be used to find all persons that match at least"
```
[the remark this answers is not in the document]
Comment (mechastriker3) [reply src/main/java/seedu/address/logic/commands/FindCommand.java:23]: that makes sense

File: src/main/java/seedu/address/model/person/Email.java [hunk 3]
```diff
@@ -56,6 +56,10 @@ public static boolean isValidEmail(String test) {
         return test.matches(VALIDATION_REGEX) || test.equals(PLACEHOLDER);
     }
 
+    public String getValue() {
```
Comment (clarlzx) [inline src/main/java/seedu/address/model/person/Email.java:59]: Perhaps can remove this since we already have toString()?

**Review by clarlzx — APPROVED** [review]
Just some things to edit (along with the things that Edeline suggested) otherwise looks good to be merged.

**Pull request merged**